src/System.Management.Automation/engine/InternalCommands.cs

```diff
@@ -351,80 +386,107 @@ protected override void ProcessRecord()
                             member = _inputObject.Members[_propertyOrMethodName];
                         }
 
-                        if (member == null)
-                        {
-                            errorRecord = GenerateNameParameterError("Name", InternalCommandStrings.PropertyOrMethodNotFound,
-                                                                     "PropertyOrMethodNotFound", _inputObject,
-                                                                     _propertyOrMethodName);
-                        }
-                        else
+                        // member is a method
+                        if ((member != null) && (member is PSMethodInfo))
```

Comment:
Why was the `null` test added? The `C#` operator `is` can be safely used with `null` values.

Reply:
D'oh! Thanks.